src/frontend_analysis/IR_analysis/InterfaceInfer.cpp

```diff
@@ -182,6 +183,14 @@ struct InterfaceInfer::interface_info
             alignment = std::max(alignment, _alignment);
          }
 
+         else if(std::regex_search(_type_name, std::regex(("bool[&*]"))) ||
+                 std::regex_search(_type_name, std::regex(("_Bool[&*]"))))
```

Comment:
Regex are costly, better use one to match both: "(_B|b)ool[&*]"

Reply:
fixed